packages/@react-aria/slider/intl/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import csCZ from './cs-CZ.json';
```

Comment:
I thought we did `import intlMessages from '../intl/*.json';` instead of index.js files?
Is this related to having translation files that aren't in this index.js?

Reply:
In this case, I import `intlMessages` into `@react-spectrum/slider/src/SliderBase.tsx` at [line 25](https://github.com/adobe/react-spectrum/pull/1148/files/c3e17c6d951594fa4f4c99312bde4a97d948a6e1#diff-892a33844e59580ce3e7173e9c524e4c6d5feb790032b9a76b26a2621b0fbe70R15), rather than `useSliderThumb`, because there doesn't seem to be a way to return the appropriate `aria-label` for the `minimum` or `maximum` thumb from `useSliderThumb`. Each thumb knows nothing about the other thumbs.

Reply:
I've refactored so that the i18n gets added in `useSliderThumb`.